### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ATOMNFT**
Added the M5 devices back in.

### CODE REVIEW

File: minigotchi-ESP32/deauth.cpp [hunk 1]
```diff
@@ -217,7 +217,11 @@ bool Deauth::select() {
       Display::updateDisplay(
           "('-')",
           "Selected AP is in the whitelist. Skipping deauthentication...");
-      Parasite::sendDeauthStatus(SKIPPING_WHITELIST);
+      < < < < < < < HEAD delay(1000);
```
Comment (dj1ch) [inline minigotchi-ESP32/deauth.cpp:220]: We should probably use the int's `Config::longDelay`(1000) and `Config::shortDelay`(500) for delays just to make the delays consistent.

**Review by dj1ch — CHANGES_REQUESTED** [review]
Looks like there were merge conflicts, make sure you remove the arrows and stuff.

File: minigotchi-ESP32/config.cpp [hunk 2]
```diff
@@ -44,7 +44,11 @@ bool Config::parasite = false;
 
 // screen configuration
 bool Config::display = true;
+<<<<<<< HEAD
+std::string Config::screen = "Define_Display_Here";
```
[the remark this answers is not in the document]
Comment (ATOMNFT) [reply minigotchi-ESP32/config.cpp]: Should be blank now.

File: minigotchi-ESP32/deauth.cpp (hunk 1 above)
Comment (ATOMNFT) [reply minigotchi-ESP32/deauth.cpp]: Delays fixed.

**Comment by ATOMNFT** [comment]
> Looks like there were merge conflicts, make sure you remove the arrows and stuff.



> Looks like there were merge conflicts, make sure you remove the arrows and stuff.

Everything should be corrected now.

**Review by dj1ch — APPROVED** [review]
lgtm

**Pull request merged**